.werft/jobs/build/deploy-to-preview-environment.ts

```diff
@@ -142,7 +142,7 @@ export async function deployToPreviewEnvironment(werft: Werft, jobConfig: JobCon
 
         try {
             werft.log(vmSlices.COPY_CERT_MANAGER_RESOURCES, 'Copy over CertManager resources from core-dev')
-            installMetaCertificates(werft, jobConfig.repository.branch, withVM, 'default', PREVIEW_K3S_KUBECONFIG_PATH, vmSlices.COPY_CERT_MANAGER_RESOURCES)
```

Comment:
I believe we want this to be blocking and not fire-and-forget. The certificates has previously been issues earlier in the job, so this is waiting for them to be ready and then copying them over.

Reply:
👍